docs/Migrating.md

```diff
@@ -1,5 +1,34 @@
 # Migrating
 
+## Migrating from ML-Agents toolkit v0.4 to v0.5
+
+### Important
+
+* The Unity project `unity-environment` has been renamed `MLAgentsSDK`.
+* The `python` folder has been renamed to `ml-agents`. It not contains two
+  packages, `mlagents.env` and `mlagents.trainers`. `mlagents.env` can be used
+  to interact directly with a Unity environment, while `mlagents.trainers`
+  contains the classes for training agents.
+
+### Unity API
+
+* Discrete Actions now have branches. You can now specify concurrent discrete
+  actions. You will need to update the Brain Parameters in the Brain Inspector
+  in all your environments.
```

Comment:
Maybe we should have a visual demonstration on how they should do the switch for the brain parameters?

Reply:
I don't think this is needed

Reply:
ok = =